go/consensus/api/light.go

```diff
@@ -42,8 +43,8 @@ type Parameters struct {
 	Height int64 `json:"height"`
 	// Meta contains the consensus backend specific consensus parameters.
 	Meta []byte `json:"meta"`
-
-	// TODO: Consider also including consensus/genesis.Parameters which are backend-agnostic.
+	// Consensus are the backend agnostic consensus parameters.
+	Consensus genesis.Parameters `json:"consensus"`
```

Comment:
```suggestion
	// Parameters are the backend agnostic consensus parameters.
	Parameters genesis.Parameters `json:"parameters"`
```
Also these should probably be placed above `Meta`.